prebuild/README.md

```diff
@@ -2,9 +2,16 @@
 
 ## Get Debian base boxes
 
+### For the 64 bit box
+
 ```bash
 vagrant box add debian/contrib-jessie64
 ```
+### For the 32 bit box
+
+```bash
+vagrant box add bento/debian-8.6-i386
```

Comment:
oh, I didn't know ! thanks for that precious information; I'll test if "bento" boxes works well for building 64b images to